lstchain/scripts/lstchain_mc_dl1_to_dl2.py

```diff
@@ -62,6 +63,15 @@ def main():
     config = replace_config(standard_config, custom_config)
 
     data = pd.read_hdf(args.datafile, key=dl1_params_lstcam_key)
+
+    # Dealing with pointing missing values. This happened when `ucts_time` was invalid.
+    if 'alt_tel' in data.columns and 'az_tel' in data.columns:
+        # make sure there is a least one good pointing value to interp from.
+        if np.isfinite(data.alt_tel).any() and np.isfinite(data.az_tel).any():
+            impute_pointing(data)
+        else:
+            data.alt_tel = - np.pi/2.
+            data.az_tel = - np.pi/2.
```

Comment:
I think that in order to avoid the interpolation between -pi/2 and correct values, the condition here to interpolate missing values should include also those values which were set to -pi/2 in dl1_to_dl2.py script:

```python
    if 'mc_alt_tel' in dl2.columns:
        alt_tel = dl2['mc_alt_tel'].values
        az_tel = dl2['mc_az_tel'].values
    elif 'alt_tel' in dl2.columns:
        alt_tel = dl2['alt_tel'].values
        az_tel = dl2['az_tel'].values
    else:
        alt_tel = - np.pi/2. * np.ones(len(dl2))
        az_tel = - np.pi/2. * np.ones(len(dl2))
```
In this way, those subruns with no alt az coordinates at all can be properly interpolated. Does this makes sense to you @vuillaut?